src/lib/operators/join_helper/join_output_writing.cpp

```diff
@@ -202,11 +202,14 @@ void write_output_segments(Segments& output_segments, const std::shared_ptr<cons
 
 namespace hyrise {
 
-std::vector<std::shared_ptr<Chunk>> write_output_chunks(
-    std::vector<RowIDPosList>& pos_lists_left, std::vector<RowIDPosList>& pos_lists_right,
-    const std::shared_ptr<const Table>& left_input_table, const std::shared_ptr<const Table>& right_input_table,
-    bool create_left_side_pos_lists_by_column, bool create_right_side_pos_lists_by_column,
-    OutputColumnOrder output_column_order, bool allow_partition_merge) {
+template <bool allow_partition_merge>
```

Comment:
Do you think the template is necessary? It's called per partition. I doubt is measurable.